From a63c1893bf09ea2a70a758374e6cbdc51c726495 Mon Sep 17 00:00:00 2001
From: Pardha Kesapragada <pkesapragada@habana.ai>
Date: Wed, 21 Apr 2021 13:24:06 +0300
Subject: [PATCH] [SW-0] correct the efuse read address register

The bootrom code incorrectly configures the
read address of the DFT_EFUSE instead of the
functional EFUSE. This patch corrects this
problem.

There is a need to reset the efuse read address
in bootrom as boot manager changes the default value
during its execution

Change-Id: I197dbb27745ce83fefbc6d4cc6a0e30bd0d56fa1
Signed-off-by: Pardha Kesapragada <pkesapragada@habana.ai>
---

diff --git a/common/efuse.c b/common/efuse.c
index 0abc3ff..59e30cf 100644
--- a/common/efuse.c
+++ b/common/efuse.c
@@ -36,7 +36,7 @@
 	/* TODO: ask suyash to test time of this function */
 	/* TODO: facto 10 of function time ~10mS*/
 	regmap_writel(EFUSE_READ_RANGE_ALL,
-		      mmPSOC_DFT_EFUSE_READ_ADDRESS);
+		      mmPSOC_EFUSE_READ_ADDRESS);
 
 	regmap_writel(1, read_reg);
 
